common/model/periods.js

```diff
@@ -0,0 +1,7 @@
+export type Period =
+  | 'today'
```

Comment:
Maybe a `TODO:` to type up `getExhibitionAndEventPromos()` with this type?

Reply:
The point would be to remove `getExhibitionAndEventPromos` all together.

Reply:
even better.